src/metabase/pulse/render/body.clj

```diff
@@ -548,29 +548,31 @@
 
   overlap = (/ overlap-width max-width) = (/ 35 59) = 0.59"
   [col-a col-b]
-  (let [[min-a min-b]    (map #(get-in % [:fingerprint :type :type/Number :min]) [col-a col-b])
-        [max-a max-b]    (map #(get-in % [:fingerprint :type :type/Number :max]) [col-a col-b])
-        non-overlapping? (or (and (< max-a min-b) (< max-a max-b))
-                             (and (> min-a min-b) (> min-a max-b)))]
-   (if non-overlapping?
-     0
-     (let [[a b c d]     (sort [min-a min-b max-a max-b])
-           max-width     (- d a)
-           overlap-width (- c b)]
-       (/ overlap-width max-width)))))
+  (let [[min-a min-b]    (map #(get-in % [:fingerprint :type :type/Number :min] 0) [col-a col-b])
+        [max-a max-b]    (map #(get-in % [:fingerprint :type :type/Number :max] 0) [col-a col-b])
+        valid-ranges?    (and min-a min-b max-a max-b)
```

Comment:
Wow, TIL that `0` is truthy for `if`s but falsey for `and`s. :exploding_head: 

Not sure I like that, Clojure…

Reply:
^^^ that's not true at all, sorry.

So, that means that `valid-ranges?` will always be `true` since the "missing" case is `(and 0 0 0 0)`, which is `0`, which is truthy.

Reply:
Oh dang, I forgot to take out the 0 in get-in. If I don't provide the fallback value there, then the min values CAN be nil, making `valid-ranges?` falsifiable.